packages/selinux/collection.yaml

```diff
@@ -11,11 +11,11 @@ packages:
       autobump.strategy: "github_tag"
   - name: "rancher"
     category: "selinux"
-    version: 0.2.rc1-1+16
+    version: "0.1.production.1"
```

Comment:
@Itxaka I'm not really sure this is semver compliant.. and besides the checksum is still the same :dancing_women:

Reply:
Checksum does nothing in the case of git strategy :/

Reply:
argh, sorry, I missed pushing the `$STRING_REPLACE` part :(